### DISCUSSION TITLE
Is the presence of the driver being tracked?

### DISCUSSION THREAD

**Discussion by goncal** [teslamate-org/teslamate#3739] [Q&A] [answered]
It would be great to be able to track at which moments there is someone sitting in the driver's seat. I have seen a "is_user_present" status being updated on MQTT, but I am unable to find a way to plot it in Grafana or to find where is that stored in the tables (I'm quite a newbie when it comes to grafana).

Background: I got a fine where a policeman claims the car was parked at the wrong spot with noone in the driver's seat, but the fact is that I was present. If I could pull data from teslamate showing that the car was sensing someone on the driver's seat it would probably help me get the fine revoked...

Can this be done? Any hints would be appreciated : )

**Comment by cwanja** [comment, marked as answer]
Short answer, no. It is tracked to publish in MQTT as you stated as well as display on the TeslaMate homepage. But it is not logged anywhere in the database to track when a user was or was not present. 

Maybe reach out to Tesla to get cabin camera footage? Our use sentry / dash cam footage to show you never left the car. Service might be able to pull logs as well.

**Reply by goncal** [reply]
Wow! Thanks for the fast and clear answer! I really appreciate it.

I'll try sending a sworn statement and adding that the police can reach out to Tesla if they need confirmation of my statement. We'll see if that works...

**Reply by cwanja** [reply]
To be clear, _you_ would probably need to reach out to Tesla to get the data. I do not think the police will do it on your behalf to prove themselves wrong 😉
